content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -429,9 +526,22 @@ same root of trust.
 To uninstall the remote cluster run the following command:
 
 {{< text bash >}}
-$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate <your original remote configuration> | \
-    kubectl delete -f -
+$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate -f istio-remote0-cluster.yaml | \
+    kubectl delete -f - --context=${REMOTE_CLUSTER_CTX}
 
 $ istioctl x create-remote-secret --context=${REMOTE_CLUSTER_CTX} --name ${REMOTE_CLUSTER_NAME} | \
     kubectl delete -f - --context=${MAIN_CLUSTER_CTX}
+
+$ unset REMOTE_CLUSTER_CTX REMOTE_CLUSTER_NAME REMOTE_CLUSTER_NETWORK
+$ rm istio-remote0-cluster.yaml
+{{< /text >}}
+
+To uninstall the main cluster run the following command:
```

Comment:
Add a comma after "cluster". (https://developers.google.com/style/commas#commas-after-introductory-words-and-phrases)

And do the same on line 526, above.

Reply:
```suggestion
To uninstall the main cluster, run the following command:
```